.snyk

```diff
@@ -5,5 +5,5 @@ ignore:
   SNYK-JAVA-IONETTY-1042268:
     - '*':
         reason: no available replacement
-        expires: 2022-04-30T00:00:00.000Z
+        expires: 2022-05-30T00:00:00.000Z
```

Comment:
The current updated version of netty still has the issue, hence the above ignore date has been extended